source/guide_rust_server.rst

```diff
@@ -0,0 +1,111 @@
+.. author:: tpraxl <kontakt@macrominds.de>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/rust-logo-blk.svg
+      :align: center
+
+###########
+Rust Server
+###########
+
+It couldn't be simpler to run a Rust web server on Uberspace.
```

Comment:
Nice to know but what is Rust? Please provide a short description as mentioned in the [style guide](https://github.com/Uberspace/lab/blob/master/STYLE.md).